### ISSUE TITLE
Endpoint for coin withdrawal?

### ISSUE THREAD

**Issue by henrydeclety** [hotbitex/hotbit.io-api-docs#3]
Hi, 

I can't find the endpoint for withdrawing assets to external addresses even though the API description says it is available. Am I missing something? or is it coming soon?

Thanks!
Henry

**Comment by henrydeclety** [comment]
Also I can't find my API Key and secret on the platform, is it coming soon too?

**Comment by Lafachief** [comment]
yeah I have the same issue here, actually I read the Doc & readme twice and I found the info that withdrawals are possible 

> .... withdrawals by using their API. 

> .... that suits themselves to check market trends, conduct transactions or withdrawals based on their application scenarios or preferences. REST API

but I couldn't find the actual endpoint for that calls :)
I am pretty sure it's not implemented yet.
